engine/src/test/java/org/camunda/bpm/engine/test/api/mgmt/BatchStatisticsQueryTest.java

```diff
@@ -257,6 +258,36 @@ public void testQueryOrderByIdDec() {
     verifySorting(statistics, inverted(batchStatisticsById()));
   }
 
+  @Test
+  public void testQueryOrderByStartTimeAsc() {
+    // given
+    helper.migrateProcessInstancesAsync(1);
+    helper.migrateProcessInstancesAsync(1);
```

Comment:
The test should ensure that the batches have actually different start times, especially on MySQL with second precision it can easily happen that the two batches have the same start time. You can use `ClockUtil` to manipulate the time in a test.